components/list/list-item-checkbox-mixin.js

```diff
@@ -7,17 +7,22 @@ export const ListItemCheckboxMixin = superclass => class extends superclass {
 
 	static get properties() {
 		return {
-			disabled: {type: Boolean },
+			disabled: { type: Boolean },
 			key: { type: String, reflect: true },
 			selectable: {type: Boolean },
 			selected: { type: Boolean, reflect: true }
 		};
 	}
 
-	// TODO: Remove grid styles after list-item-generic is created
 	static get styles() {
 		return [ checkboxStyles, css`
-			.d2l-checkbox-action { height: 100%; }
+			.d2l-checkbox-action {
+				height: 100%;
+				cursor: pointer;
+			}
+			:host-context([disabled]) .d2l-checkbox-action {
```

Comment:
I think this should/could just be:
```
:host([disabled]) .d2l-checkbox-action {
    ...
}
```

Aside from that, I don't think we can use `:host-context` going forward.  There's a bit of discussion about in the `webcomponentsjs` repo, and while I haven't searched for the actual meeting notes on it, as far as I know, there is still no consensus or intention on Apple's part to support it.

Reply:
Good to know - I will just pass the disabled state down to the layout component from the parent in that case.

Reply:
I don't think it will need to be passed though, since `disabled` is a property of a mixin, and therefor on the `host`.